frontend/ui/widget/textviewer.lua

```diff
@@ -69,26 +73,35 @@ local TextViewer = InputContainer:extend{
     find_centered_lines_count = 5, -- line with find results to be not far from the center
 }
 
-function TextViewer:init()
-    -- calculate window dimension
+local text_categories = {
```

Comment:
I use the word "category" when I introduced the idea, because it's the french word that came to my mind :) Dunno if it's really the best word for the setting and the variables. Sounds a bit odd reading that in English - for something that is more a kind or a type. @Frenzie @NiLuJe ? We will write and read:
```lua
    textviewer = TextViewer:new{
        title = _("Selection HTML"),
        text = html,
        text_category = "code", -- or "file_content", or "general"
```
Does this reads ok to you?

Reply:
Re-pinging @Frenzie @NiLuJe

Reply:
I think type would be the most obvious and variety would probably read a little more natural than category but I'm fine with category too.

Reply:
Yup, my first thought would have been `type` too, but this is fine ;).